### PR TITLE
second attempt to trying to fix merge errors

### PR DESCRIPTION
**Pull request by Bobafish**

### CODE REVIEW

**Comment by Bobafish** [comment]
Created new pull request. Same test as #9, but set the package-lock.json to be up to date so I don't get merge conflicts

File: package.json [hunk 1]
```diff
@@ -9,13 +9,12 @@
...
     "test": "npm run test:unit; npm run test:integration; npm run test:e2e",
-    "test:unit": "jest --testPathIgnorePatterns=bar",
-    "test:integration": "jest bar",
+    "test:unit": "jest --testPathIgnorePatterns=bar, chartBuilder",
+    "test:integration": "jest bar chartBuilder",
     "test:e2e": "start-server-and-test start:e2e http://localhost:8080 cy:run"
   },
   "devDependencies": {
     "@testing-library/cypress": "^10.0.1",
-    "start-server-and-test": "^2.0.3",
```
Comment (jwevans21) [inline package.json:18]: This is needed for the e2e tests. If not included it will either cause merge conflicts or errors when running tests

**Review by jwevans21 — CHANGES_REQUESTED** [review]
Looks good. Just the one fix needed.

To handle the merge conflicts with the `package-lock.json` you can on your computer:
- `git switch <your feature branch>`
- `git merge main`
- Fix conflicts in other files
- Remove the `package-lock.json`
- Run `npm install`
- Add and commit the merge changes
- When merging on GitHub use the changes from your current branch for the `package-lock.json`

Let me know if you have any questions.

File: package.json (hunk 1 above)
Comment (Bobafish) [reply package.json:18]: The start-server-and-test is still in the dev dependencies. Idk why, but it ended up getting arranged to be on a later line (line 26)

**Review by Aic01 — APPROVED** [review]

**Comment by Aic01** [comment]
I think it is good once you finished Jacob's request

File: package.json (hunk 1 above)
Comment (jwevans21) [reply package.json:18]: Alright well that explains why the tests still passed

**Review by jwevans21 — APPROVED** [review]

**Pull request merged**